spec/koans.js

```diff
@@ -155,30 +155,30 @@ describe('a template string, is wrapped in ` (backticks) instead of \' or ". ',
   describe('can evaluate variables, which are wrapped in "${" and "}"', function() {
 
     it('e.g. a simple variable "${x}" just gets evaluated', function() {
-      let evaluated = `x=x`
-      //expect(evaluated).toBe('x=' + x);
+      let evaluated = `x=${x}`
+      expect(evaluated).toBe('x=' + x);
     });
 
     it('multiple variables get evaluated too', function() {
-      var evaluated = `x+y`;
-      //expect(evaluated).toBe(x + '+' + y);
+      var evaluated = `${x}+${y}`;
```

Comment:
Trata de separar un poquito el códigoo para que se lea mejor
```suggestion
      var evaluated = `${ x } + ${ y }`;
```